integr8ly/installation
# Update release script to reset upgrade playbook on final release

## Additional Information
Update release script to reset upgrade playbook on final release (Anything that isn't a CR or ER)

## Verification Steps

* Checkout this branch and create a new test release branch from it and push to your fork (origin in my case):
`git checkout -b v9.9 && git push origin v9.9`
* Create a new rc1, and verify it creates a new rc1 tag from the HEAD of the v9.9 branch and upgrade playbook isn't modified:
`REMOTE=origin ./scripts/release.sh -b v9.9 -r release-9.9.0-rc1`
* Create a new rc2, and verify it creates a new rc2 tag from the HEAD of the v9.9 branch and upgrade playbook isn't modified:
`REMOTE=origin ./scripts/release.sh -b v9.9 -r release-9.9.0-rc2`
* Create a new final release, and verify it creates a new release tag from the HEAD of the v9.9 branch and upgrade playbook and variables are reset:
`REMOTE=origin ./scripts/release.sh -b v9.9 -r release-9.9.0`

### Upgrade Verification
N/A

/cc @maleck13
